modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -46,12 +46,16 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-darwin/l
 curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
 ----
 
-. Change file permissions for the `rtfctl` command-line utility:
+. Change file permissions for the `rtfctl` command-line tool:
 +
 ----
 sudo chmod +x rtfctl
 ----
 
+For Runtime Fabric on VMs / Bare Metal,  you must run `rtfctl` from a controller node. The tool detects the required configuration to manage the cluster. 
+
+For Runtime Fabric on Self-Managed Kubernetes, `rtfctl` consumes kubeconfig files as described in https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/[Organizing Cluster Access Using kubeconfig Files^]. Defer to this method to configure which cluster you want to manage.
```

Comment:
I'm not quite sure what you mean by "Defer to this method". Would "Use this method" work?